src/common/InputWithIcon/InputWithIcon.tsx

```diff
@@ -1,19 +1,31 @@
 import React from 'react';
 import "./InputWithIcon.css";
+import PasswordToggle from "../PasswordToggle";
 
 type Props = {
     disabled?: boolean,
     value?: string,
     onChange?: (event: any)=>void,
     icon: string,
     placeholder?: string,
+    isPassword: boolean,
```

Comment:
Here the `isPassword` prop could be optional (like this: `isPassword?: boolean`).
 
This way, if you don't need the PasswordToggle you can just avoid passing the prop, and this will still work:
`props.isPassword? "Password": "input"`